static/app/views/releases/drawer/releasesDrawerDetails.spec.tsx

```diff
@@ -0,0 +1,222 @@
+}));
+
+// Mock the hooks
+jest.mock('sentry/utils/useLocation', () => ({
+  useLocation: jest.fn().mockReturnValue({
+    pathname: '/releases/',
+    query: {},
+  }),
+}));
+
+jest.mock('sentry/utils/useNavigate', () => ({
+  __esModule: true,
+  useNavigate: jest.fn().mockReturnValue(jest.fn()),
+}));
+
+describe('ReleasesDrawerDetails', () => {
+  const defaultProps = {
+    release: 'test-release',
+    projectId: undefined,
+  };
+
+  const organization = OrganizationFixture();
+
+  beforeEach(() => {
+    MockApiClient.clearMockResponses();
+    jest.clearAllMocks();
+
+    ProjectsStore.reset();
+    ProjectsStore.loadInitialData([
+      ProjectFixture({id: '1', slug: 'test-project'}),
+      ProjectFixture({id: '2', slug: 'test-project-2'}),
+    ]);
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/releases/test-release/meta/',
+      method: 'GET',
+      body: ReleaseMetaFixture(),
+    });
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/releases/test-release/',
+      method: 'GET',
+      body: ReleaseFixture(),
+    });
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/releases/test-release/deploys/',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/projects/org-slug/test-project/releases/test-release/repositories/',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/repos/',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/users/',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/issues/?end=2017-10-17T02%3A42%3A00Z&groupStatsPeriod=auto&limit=10&project=2&query=first-release%3Atest-release&sort=freq&start=2017-10-17T02%3A41%3A00Z',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/issues/?end=2020-03-24T02%3A04%3A59Z&groupStatsPeriod=auto&limit=10&project=2&query=first-release%3Atest-release&sort=freq&start=2020-03-23T01%3A02%3A00Z',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/issues/?end=2017-10-17T02%3A42%3A00Z&groupStatsPeriod=auto&limit=10&project=1&query=first-release%3Atest-release&sort=freq&start=2017-10-17T02%3A41%3A00Z',
+      method: 'GET',
+      body: [],
+    });
+  });
+
+  afterEach(() => {
+    MockApiClient.clearMockResponses();
+    jest.clearAllMocks();
+  });
+
+  it('renders content when projectId exists', async () => {
+    render(<ReleasesDrawerDetails {...defaultProps} projectId="1" />, {organization});
+
+    await waitFor(() => {
+      expect(screen.queryByTestId('loading-indicator')).not.toBeInTheDocument();
+    });
+
+    expect(screen.getByText('Details')).toBeInTheDocument();
```

Comment:
```suggestion
    expect(await screen.findByText('Details')).toBeInTheDocument();
```

does the test work if we await the text instead?